steps/cocoapods-dependency-check/0.1/step.yml

```diff
@@ -0,0 +1,73 @@
+title: cocoapods-dependency-check
+summary: |
+  OWASP dependency analysis for iOS / CocoaPods
+description: |
+  Uses jeremylong/DependencyCheck to search Podfile.lock for CVE vulnerabilites.
+
+  The generated HTML document will be placed into BITRISE_DEPLOY_DIR by default.
+  Remember to run the "Deploy to bitrise.io" step in order to persist artifacts.
+website: https://github.com/johnjcsmith/bitrise-step-cocoapods-dependency-checker
+source_code_url: https://github.com/johnjcsmith/bitrise-step-cocoapods-dependency-checker
+support_url: https://github.com/johnjcsmith/bitrise-step-cocoapods-dependency-checker/issues
+published_at: 2018-08-02T16:47:02.19911406+09:30
+source:
+  git: https://github.com/johnjcsmith/bitrise-step-cocoapods-dependency-checker.git
+  commit: f6a389a794df11e6923f335c4d219650e7060d41
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
```

Comment:
If this step is only for iOS than remove please the `- ubuntu-16.04` from `host_ost_tag`.